docs/contributing/cursorless-in-neovim.md

```diff
@@ -0,0 +1,51 @@
+# Cursorless in Neovim
+
+This document describes how to get set up to work on the Cursorless neovim plugin.
+
+## Initial setup
+
+### 1. Follow the initial contributor setup guide
+
+See [CONTRIBUTING.md](./CONTRIBUTING.md). In particular:
+
+- Build the VSCode Cursorless extension
+- Run the VSCode Cursorless extension tests
```

Comment:
What does it mean to "build the vscode cursorless extension"? And why are these steps necessary?

Reply:
you removed this after we discussed about it